ledger/ledger-on-sql/src/test/lib/scala/com/daml/ledger/on/sql/SqlLedgerReaderWriterIntegrationSpecBase.scala

```diff
@@ -26,13 +26,12 @@ abstract class SqlLedgerReaderWriterIntegrationSpecBase(implementationName: Stri
       metrics: Metrics,
   )(implicit loggingContext: LoggingContext): ResourceOwner[ParticipantState] =
     new SqlLedgerReaderWriter.Owner(
-      ledgerId,
-      participantId,
-      metrics,
+      ledgerId = ledgerId,
+      participantId = participantId,
+      metrics = metrics,
       engine = Engine.DevEngine(),
-      jdbcUrl(testId),
+      jdbcUrl = jdbcUrl(testId),
       resetOnStartup = false,
-      // Using a weak random source to avoid slowdown during tests.
-      seedService = SeedService(Seeding.Weak),
+      logEntryIdAllocator = LogEntryIdAllocator.Random,
```

Comment:
Not really related to this PR, but I am worried that it's not apparent anywhere in the code that this impl. is used only for testing (is it?).

Reply:
Kind of; depends how you define "testing". This code is only used by _ledger-on-sql_'s own `Main` entrypoint (not Sandbox), and so therefore it's only used in tests.

As this implementation is only used in _ledger-on-sql_, I'll just move it there and rename it accordingly.

Reply:
Moved to _ledger-on-sql/app_.

Reply:
OK, now I am only worried that it's not apparent anywhere in the code that `ledger-on-sql` is only used for testing (is it?) :)

Reply:
_ledger-on-sql/app_ is only used for testing; that's where I moved it.

Reply:
Pushed a _README.md_.